Flick/Supporting/Constants.swift

```diff
@@ -10,6 +10,18 @@ import Foundation
 
 struct Constants {
 
+    struct Collaboration {
+        static let onlyICanEdit = "Only I can edit"
+        static func numCanEdit(num: Int) -> String {
+            return "\(num) can edit"
+        }
+    }
+
+    struct Privacy {
+        static let anyoneCanView = "Anyone can view"
+        static let onlyICanView = "Only I can view"
```

Comment:
yay, good use of the constants file haha. ive been so lazy with making things into variables...